extensions/amp-story/1.0/amp-story.css

```diff
@@ -210,8 +210,12 @@ amp-story-page[active] {
  * not be automatically laid out. Max distance is set to 2 (next 2 pages) since
  * we don't want to schedule any further pages. */
 
+amp-story-page:not(:first-of-type):not([distance]):not([active]) {
+  transform: translateY(1000vh) !important;
```

Comment:
Just a reminder to add a comment here or in `amp.css` that references these two, in case we make a change in the future - so we don't forget to update it in `amp.css`.